release_notes.md

```diff
@@ -1,13 +1,3 @@
 # What's New
-## Asset Module Enhancements
-- Added a new picklist and a field **Asset Risk** to the **Asset** module to denote the risk level of the asset based on the highest Vulnerability severity associated with an asset.
-
->**Note:** This field has not been added to the Asset Detailed view. You can add this by editing the Detailed View Template.
-
-## Playbook Enhancements
-
-- Modified the **Scenario -  Ingest Scan Records** playbook
-  - A playbook now marks **Asset Risk** value based on the highest Vulnerability severity associated with the asset
-- Modified following response playbooks so that the playbooks now fetch and create alerts for the assets whose **Asset Risk** is **Critical** or **High**
-  - Fetch High-Risk Assets for Scan
-  - Create or Update Alerts for High-Risk Asset
+## Module Enhancements
+- Added a new CVEs module
```

Comment:
Should we also mentioned about its entry in navigation menu, new picklists, etc?

Reply:
Updated release notes as suggested